### ISSUE TITLE
Unable to Scan QR Code 

### ISSUE THREAD

**Issue by netslutter** [yusufusta/WhatsAsena#58]
Unable to scan qr code in whatsapp web ! , check the to attachment

https://user-images.githubusercontent.com/79210553/122213436-94c95100-cec6-11eb-97fe-5e96ad77890d.mp4

**Comment by teshan200** [comment]
same

**Comment by XKeyScore147** [comment]
Use This :
https://replit.com/@phaticusthiccy/WhatsAsena-QR

**Comment by netslutter** [comment]
> Use This :
> https://replit.com/@phaticusthiccy/WhatsAsena-QR

![image](https://user-images.githubusercontent.com/79210553/122577625-a2bad580-d070-11eb-84ab-052d364b4b38.png)

**Comment by XKeyScore147** [comment]
Yes Yes , That is completely normal. Just Copy the WhatsAsena Session and use it to deploy the bot
